app/Http/Controllers/AdminController.php

```diff
@@ -26,4 +28,77 @@ public function users()
         $users = User::all(); // Retrieve all registered users
         return view('admin.users', compact('users'));
     }
+
+    public function updateUser(Request $request, $id)
```

Comment:
- add comment method above 
- check alignments inside the function
- add insertion in the model then add error in the controller